testing/run-scripts/run_cloud.sh

```diff
@@ -64,7 +65,17 @@ then
     fi
     BANNER=`docker exec uproxy-sshd cat /banner`
   else
-    BANNER=`hostname`
+    # DigitalOcean's metadata API can tell us the region in which
+    # the machine is located:
+    #   https://developers.digitalocean.com/documentation/metadata/#metadata-api-endpoints
+    BANNER=`curl -s -m 2 http://169.254.169.254/metadata/v1/region`
+    if [ ! -z "$BANNER" ]
+    then
+      BANNER=`echo "$BANNER"|sed 's/ams./Netherlands/;s/sgp./Singapore/;s/fra./Germany/;s/tor./Canada/;s/nyc./New York, USA/;s/sfo./San Francisco, USA/;s/lon./UK/'`
+      BANNER="$BANNER (DigitalOcean)"
```

Comment:
If this droplet is administered by an organization which maintains more than one uProxy Cloud Server in this same region, they will both get the same name. Is it worth tacking something else onto the banner to make name collisions less likely? (maybe the current time? first few characters of this instance's public key? or even `$RANDOM`?) Maybe this is too much spit and polish. :)